website/project/model.py

```diff
@@ -3342,9 +3340,6 @@ def add_unregistered_contributor(self, fullname, email, auth, send_email='defaul
     def add_contributor_registered_or_not(self, auth, user_id=None, full_name=None, email=None, send_email='false',
                                           permissions=None, bibliographic=True, index=None, save=False):
 
-        if send_email != 'false' and not throttle_period_expired(auth.user.email_last_sent, settings.API_SEND_EMAIL_THROTTLE):
```

Comment:
@brianjgeiger, I'm removing the previously added throttling from the model here.